common/src/main/java/uk/nhs/adaptors/common/util/CodeableConceptUtils.java

```diff
@@ -29,6 +30,16 @@ public static CodeableConcept createCodeableConcept(String code, String system,
         return codeableConcept;
     }
 
+    public static CodeableConcept createCodeableConceptWithDoubleCoding(String code, String system, String display,
```

Comment:
This should be a more specific name for the method, as it specifically creates a codeable concept with the oid for "EhrRequestAckCode", whereas the name suggests it is more generic